common/constants.ts

```diff
@@ -115,13 +114,29 @@ export const staticEnvironmentPayload = {
       Uuid: '8ceea3cd-7608-4428-8d6b-99f2acc80ce3',
       Name: 'Shuttle-8',
     },
+    {
+      Uuid: 'd2962032-37e4-43f5-ad9b-15535c5555d6',
+      Name: 'Shuttle-9',
```

Comment:
asked Luc for updated Shuttle uuids & will update them for the latest ones before merging